Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -314,9 +378,260 @@ def isImportsOnlyFile (lines : Array String) : Bool :=
 
 end
 
+
+namespace unicodeLinter
+
+/-- Hashable instance for use with the unicodeLinter -/
+local instance instHashableChar : Hashable Char where
+  hash c := c.val.toUInt64
+
+/-- Allowed whitespace characters -/
+def ASCII.whitespace (c : Char) := #[' ', '\n'].contains c
```

Comment:
I would call this e.g. `allowedWhitespace` so as to avoid confusion with the concept "all ASCII whitespace".

Reply:
I renamed it.